tests/test_integration.py

```diff
@@ -13,18 +13,31 @@
 
 NATIVE_TOXENV = f"py{sys.version_info[0]}{sys.version_info[1]}"
 NATIVE_EXECUTABLE = str(pathlib.Path(sys.executable).resolve())
-TOX_INI = pathlib.Path(__file__).parent / "tox.ini"
-DOT_TOX = pathlib.Path(__file__).parent / ".tox"
+FIXTURES_DIR = pathlib.Path(__file__).parent / "fixtures"
```

Comment:
I don't really understand this change. Why has the tox.ini moved from the same directory as `__file__` to the subdirectory `fixtures`?

Reply:
As far as I can tell it's just a cosmetic change?

Reply:
Previously, the file was just there and it was used by the invoked tox via a config switch.
Now, it lives in a dedicated fixtures folder and is copied to a temporary directory, used from there.
You are right that this change was not strictly necesery, hence might be considered cosmetic. I usually put files used by pytest fixtures to a fixtures folder.

Reply:
Ok, sounds good.